lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -0,0 +1,315 @@
+from pathlib import Path
```

Comment:
Note that on a next pass, we can refactor a lot of this repeated code at the type of job logic split into a common shared class between all types of jobs. Since this is a big change, I'm only doing a straight duplication to make sure it all works first.

Reply:
That's perfectly fine! I myself am experimenting with inference batch jobs by simply modifying the current eval code, and I see that a good part of it is common.
I'd say let's do this round, see what differs between different job calls, then finally factor this out and make it parametric wrt the job type.